docs/mining/proof-of-work.md

```diff
@@ -69,30 +69,21 @@ GPU drivers usually contain the libraries needed for mining.  If you have diffic
 
 ## **<i class="fa fa-download"></i> Select and Download Mining Software **
 
-> Official Decred Mining Software Builds
+### Official Decred Miner (gominer)
 
-#### **[<i class="fa fa-github"></i> gominer](https://github.com/decred/gominer)**
-
-It is recommended that everyone use gominer for Decred. This is the easiest miner to setup and start mining. Currently, the latest version of gominer is **[<i class="fa fa-github"></i>v1.0.0](https://github.com/decred/gominer/releases/)** and the official binaries can be downloaded from **[https://github.com/decred/decred-binaries/releases/tag/v1.0.0](https://github.com/decred/decred-binaries/releases/tag/v1.0.0)**
+It is recommended that everyone use gominer for Decred. This is the easiest miner to setup and start mining. Currently, the latest version of gominer is **<i class="fa fa-github"></i> [v1.0.0](https://github.com/decred/gominer/releases/)** and the official binaries can be downloaded from **[https://github.com/decred/decred-binaries/releases/tag/v1.0.0](https://github.com/decred/decred-binaries/releases/tag/v1.0.0)**
```

Comment:
I realize it always said this, but do we really want to say it so strongly?  The other miners are good too, gominer is just the one we can actually help with and make changes to.  Maybe we should say 'encouraged' instead of 'recommended'?  I agree that it is easiest to get started with though.  So not really sure.  My point is just that this sounds too strongly in favor of gominer instead of the others (we don't actively discuourage the other miners, we just encourage gominer).  Any thoughts?